crates/adapters/src/controller/mod.rs

```diff
@@ -3829,11 +3831,19 @@ impl InputProbe {
         connector_config: &ConnectorConfig,
         controller: Arc<ControllerInner>,
     ) -> Self {
+        // Create quota: 10 requests within 10 seconds
+        let quota = governor::Quota::with_period(Duration::from_secs(10))
+            .unwrap()
+            .allow_burst(nonzero_ext::nonzero!(10u32));
+
+        let rate_limiter = SamplingRateLimiter::new(quota, 5);
```

Comment:
what would be sane default for quota? ( like at max N reqs in T secs )

and also for sampling threshold ( say if it is 50, and we already have suppressed 50 logs within T secs, then we _allow_ 51st log for sampling ) maybe something like 100?

Reply:
In the past, I've used leaky-bucket rate limiters for log messages, with typical limits on the order of 5-10 per minute, with a burst size that's about the same or a bit bigger.

It's a good idea to track and log the number of messages that were suppressed when the next one is printed; here's an example of what we did in another project: https://github.com/openvswitch/ovs/blob/main/lib/vlog.c#L1363-L1366